### PR TITLE
Organ Spilling Rebalance

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

This isn't an organ spill rebalance but rather a bugfix that fixes sexual organs being spilled out on disembowelment. The title exists so that future employers don't look at my github history and see "Fixes anuses, vaginas, penises, testicles spilling out during disembowelment" on my git history.

I did not test this yet because I need to go to bed and I just spent the last 10 minutes fixing my fucked git repo. But it compiles, so it works:tm:.

actually im not even sure if it compiles i did not even bother to compile it

## Changelog

:cl: BurgerBB
fix: Fixes sexual organs being spilled out on disembowelment.
balance: Removes cock checking.
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
Bro didn't even compile his code and straight up deleted upstream's code. So incredibly based

File: tgstation.dme [hunk 1]
```diff
@@ -5441,11 +5441,11 @@
 #include "interface\interface.dm"
 #include "interface\menu.dm"
 #include "interface\stylesheet.dm"
+#include "interface\skin.dmf"
```
Comment (nevimer) [inline tgstation.dme:5444]: huh

File: tgstation.dme [hunk 2]
```diff
@@ -5635,7 +5635,6 @@
 #include "modular_skyrat\master_files\code\modules\mob\living\emote_popup.dm"
 #include "modular_skyrat\master_files\code\modules\mob\living\examine_tgui.dm"
 #include "modular_skyrat\master_files\code\modules\mob\living\living_defines.dm"
-#include "modular_skyrat\master_files\code\modules\mob\living\carbon\death.dm"
```
Comment (nevimer) [inline tgstation.dme:5638]: why, just comment block out this file we arnt getting this as a merge conflict later

**Review by nevimer — CHANGES_REQUESTED** [review]

File: tgstation.dme (hunk 1 above)
Comment (BurgerLUA) [reply tgstation.dme:5444]: byond did that

**Comment by BurgerLUA** [comment]
image

wow it actually works on first compile, but I did the requested changes

**Pull request merged**